packages/http/src/mocker/generator/HttpParamGenerator.ts

```diff
@@ -51,7 +52,7 @@ function pickStaticExample(
   );
 }
 
-export function generate(param: IHttpParam): Option.Option<unknown> {
+export function generate(param: { schema?: JSONSchema4 | JSONSchema6 | JSONSchema7; examples?: Array<INodeExample | INodeExternalExample> }): Option.Option<unknown> {
```

Comment:
Then I guess you can just do

```suggestion
export function generate(param: IHttpParam | IHttpContent): Option.Option<unknown> {
```

So that only the fields in common can be used inside this function as long you do not use a type guard.

Reply:
Good point, updated 👍